typings/xpath-to-css.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'xpath-to-css' {
```

Comment:
This should be removed, and I should do a pass to cull any now-unused dependencies too.